tests/main/snapctl/task.yaml

```diff
@@ -0,0 +1,80 @@
+summary: Check that `snapctl` can be run from within hooks
+
+systems: [-ubuntu-core-16-64]
+
+prepare: |
+    echo "Ensure we have a core snap with snap run"
+    $SPREAD_PATH/tests/lib/reset.sh
+    snap install --channel=candidate ubuntu-core
```

Comment:
This (6,7,8) should no longer be needed, the current stable ubuntu-core has `snap run` now.